src/Catel.Core/Catel.Core.Shared/Collections/ArrayShim.cs

```diff
@@ -18,9 +18,9 @@ namespace Catel.Collections
     public static class ArrayShim
     {
 #if ARRAY_EMPTY_MISSING
-        internal static class EmptyArray<T>
```

Comment:
Note sure if we can make this change. Shouldn't this class be available to the outside so everyone can use the ArrayShim? Or are there other members that people can still use?

Reply:
Fixed